examples/slurm/README.md

```diff
@@ -0,0 +1,218 @@
+# SLURM-based HPC Cluster Setup Guide
+This guide will help you deploy ServerlessLLM on a SLURM-based HPC (High Performance Computing) cluster. Please make sure you have installed the ServerlessLLM following the [installation guide](https://serverlessllm.github.io/docs/stable/getting_started/installation#installing-with-pip) on all machines.
+## Some Tips about Installation
+- **Both installation and build require an internet connection**. If you want to do installation or build tasks on a job node, it is better to make sure the port 443 is accessible.
+- We recommend using [installing with pip](https://serverlessllm.github.io/docs/stable/getting_started/installation#installing-with-pip) if there is no CUDA driver on login node.
+If you want to [install from source](https://serverlessllm.github.io/docs/stable/getting_started/installation#installing-from-source), please make sure CUDA driver available on the login node.
```

Comment:
Maybe changing it to "on the node you want to install" is better than directly mentioning the login node. Same in line 5.

Reply:
No problem, it is better.